.vscode/launch.json

```diff
@@ -0,0 +1,68 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Start Rails server",
+            "type": "Ruby",
+            "request": "launch",
+            "cwd": "${workspaceRoot}/my-sms",
+            "program": "${workspaceRoot}/my-sms/bin/rails",
+            "args": [
+                "server"
+            ]
```

Comment:
Do these configs work with containers? I can see you've mentioned the vscode remote development extension in your README, so I assume this is working with that, but would have thought this config would be attempting to start the rails server on your machine, rather than in the container.

Reply:
Using this vscode runs the config inside the container, launching both the debugger and server. I'd have to specify a container/local host if I was already running the debugger and wanted to connect to it.

Reply:
That's really neat. I might need to look into setting up a similar config myself.